mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -73,10 +130,10 @@ private[ml] trait OneVsRestParams extends PredictorParams {
 @Since("1.4.0")
 @Experimental
 final class OneVsRestModel private[ml] (
-    @Since("1.4.0")  override val uid: String,
-    @Since("1.4.0") labelMetadata: Metadata,
+    @Since("1.4.0") override val uid: String,
+    @Since("1.4.0") val labelMetadata: Metadata,
```

Comment:
This should be private[ml].  The Since tag was a mistake and can be removed since it does not show up in the public API.